ci-operator/step-registry/windows/e2e/operator/test/windows-e2e-operator-test-ref.yaml

```diff
@@ -5,6 +5,6 @@ ref:
   resources:
     requests:
       cpu: 1000m
-      memory: 600Mi
+      memory: 1Gi
```

Comment:
Doesn't this need to be reflected in other places like https://github.com/openshift/release/blob/master/ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml?